.github/workflows/ci.yml

```diff
@@ -710,7 +792,7 @@ jobs:
     needs:
       - set-condition
       - parachain-ts-test
-      - tee-test
+      - tee-single-worker-test
```

Comment:
is it necessary to add `tee-multi-worker-test` dependency here?

Reply:
Yea I believe you need to add `tee-multi-worker-test` to the `needs` here

Reply:
I didn't add it here because `tee-multi-worker-test` has `continue-on-error: true` - the output of that job shouldn't affect final outcome of workflow. 

Once we remove `continue-on-error` we will add it here. What do you think?

Reply:
That works too